change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW

```diff
@@ -709,14 +712,25 @@ PROCEDURE GETNAMELOCS
 ** information for allowed name types and levels. Only
 ** called if old address matching is enabled.
 *]
+ FOR EACH NAME WITH (NAME:TYPE=0)
+  DO
+   OLDSTREET=NAME:STREET
+   OLDEXTRA=NAME:EXTRAADDRESS
+   OLDCITY=NAME:CITY
+   OLDSTATE=NAME:STATE
+   OLDZIP=NAME:ZIPCODE
+  END
  IF MATCHACCTNAMES=TRUE THEN
   DO
    FOR EACH NAME WITH ((NAME:EXPIRATIONDATE=DATENULL OR
```

Comment:
Should the check for name record qualifications include the NAME:EFFECTIVEDATE?  Maybe not.... just want to make sure it was considered.